README.md

```
@@ -370,6 +370,50 @@ use MaikSchneider\TcaApi\Filter\WordStartFilter;
+```php
+use MaikSchneider\TcaApi\Filter\ExactFilter;
+
+'filters' => [
+    'color_id.name'           => ExactFilter::class,  // one FK hop     → the colour's name
+    'categories.title'        => ExactFilter::class,  // one MM hop     → a category's title
+    'related_items.name'      => ExactFilter::class,  // one inline hop → an inline child's name
+    'categories.parent.title' => ExactFilter::class,  // two hops       → a category's parent's title
+],
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)